src/Cli/dotnet/BuiltInCommandsCatalog.cs

```diff
@@ -125,6 +125,7 @@ public static class BuiltInCommandsCatalog
             ["sdk"] = new BuiltInCommandMetadata
             {
                 Command = SdkCommand.Run,
+                DocLink = "https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-sdk-check"
```

Comment:
Should these be aka.ms links in case pages get shuffled around?

Reply:
Good idea, just updated.